src/api/main/SignupForm.js

```diff
@@ -177,6 +177,20 @@ function signup(mailAddress: string, pw: string, registrationCode: string, isBus
 	})
 }
 
+/**
+ * Accepts multiple formats for a time of day and always returns 12h-format with leading zeros.
+ * @param captchaInput
+ * @returns {string} HH:MM if parsed, empty string otherwise
+ */
+export function parseCaptchaInput(captchaInput: string): string {
```

Comment:
I would rather return `null` so that it's visible on type level that there are two cases